pkg/plugins/golang/v3/api.go

```diff
@@ -107,30 +105,32 @@ After the scaffold is written, api will run make on the project.
 }
 
 func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
-	fs.BoolVar(&p.runMake, "make", true, "if true, run make after generating files")
+	p.options = &goPlugin.Options{}
+	fs.StringVar(&p.options.Group, "group", "", "resource Group")
+	p.options.Domain = p.config.Domain
```

Comment:
```suggestion
```
We ought to need to set that here. And then, if we remove this line all stop to work. If we try to add this logic inside of the newResource where it shows more appropriated the tooling will still not working which lead to me think that it is a workaround to make the changes made work which raises so concerns about the logic in place.

Reply:
Why is setting an options field from a config field a design failure? Options holds everything that will be needed to create a resource, and the domain is needed. It also offers a clear rute for when we provide this as an flag for users, which will happen once we accept external resources.